subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -339,6 +339,16 @@ Many of these options can be specified in `gradle.properties` so command-line fl
 `--build-cache`, `--no-build-cache`::
 Toggles the <<build_cache.adoc#build_cache, Gradle build cache>>. Gradle will try to reuse outputs from previous builds. _Default is off_.
 
+`--configuration-cache`, `--no-configuration-cache`::
+Toggles the <<configuration_cache.adoc#sec:config_cache,Configuration Cache>>. Gradle will try to reuse the build configuration from previous builds. _Default is off_.
```

Comment:
ah, @eskatos I was trying to link to the top of the page. In adoc it has `[[config_cache]]`, how do I just link to the page without linking a section?
```suggestion
Toggles the <<configuration_cache.adoc,Configuration Cache>>. Gradle will try to reuse the build configuration from previous builds. _Default is off_.
```
?

Reply:
Figured it out, ran this locally a few times
```
gradlew :docs:checkDeadInternalLinks
```
and experimented, the problem was that I thought the syntax is `#sec:somesection`, but it's just `#somesection` and `sec:` is a Gradle convention in some files to signify a section.

Reply:
https://github.com/gradle/gradle/pull/18955/commits/5b1e6416698bf76ec8d2d1f248b3c078023e4b21